lib/airbrake.rb

```diff
@@ -19,3 +19,28 @@
 require 'airbrake/resque/failure' if defined?(Resque)
 require 'airbrake/sidekiq/error_handler' if defined?(Sidekiq)
 require 'airbrake/delayed_job/plugin' if defined?(Delayed)
+
+##
+# This module defines the Airbrake API and serves as a namespace for other
+# classes and modules.
+##
```

Comment:
Nitpick: we don't end comments with `##`, so I suggest to remove them (but keep `##` in the beginnings), so it looks consistent.